library/src/test/java/bio/terra/landingzone/stairway/flight/create/CreateLandingZoneFlightTest.java

```diff
@@ -30,22 +32,6 @@ class CreateLandingZoneFlightTest {
 
   @Mock private LandingZoneFlightBeanBag mockApplicationContext;
 
-  @Test
-  void testInitializationWhenIsNotAttaching() {
```

Comment:
Because this test has been removed, the other test code in this class can be simplified: there is now only one case, "attaching".

Reply:
There isn't a corresponding test for `NotAttaching` here, because that would cause network calls to Azure to initialize the `ArmManagers` when setting up the resource steps.

Because of how stairway works, and the heavy requirement on difficult to mock resources, however we structure the flight, there's going to be one of 
1. a runtime cost
2. a code-structure or behavior issue issue
3. a part that's difficult to unit test

I attempted to move this test case to the integration test, but there were issues with stairway setup and resource initialization.

My calculation was that we have an E2E test that creates all the resources, and that wouldn't work at all if the mainline code path was broken.

Reply:
This discussion around `ArmManagers` and the difficulty they cause with testing makes me think a service layer would help here. Much of our step code is direct calls to the Azure SDK. FWIW, I've been pushing us away from that in WSM toward making steps dumber and delegating calls to helpers/services/whatever and having a cleaner separation of concerns.